plugins/spender/multifundchannel.c

```diff
@@ -1504,16 +1504,6 @@ perform_fundpsbt(struct multifundchannel_command *mfc, u32 feerate)
 				tal_fmt(tmpctx, "%zu", startweight));
 	}
 
-	/* If we've got v2 opens, we need to use a min weight of 110. */
-	/* BOLT-f53ca2301232db780843e894f55d95d512f297f9 #3:
-	 * The minimum witness weight for an input is 110.
-	 */
-	if (dest_count(mfc, OPEN_CHANNEL) > 0) {
-		json_add_string(req->js, "min_witness_weight",
-				tal_fmt(tmpctx, "%u", 110));
-	}
-
-
```

Comment:
I don't think you can just delete this?

Reply:
ok this one is obviously true as this is an *empty* input, putting back